src/snowflake/snowpark/dataframe.py

```diff
@@ -450,6 +450,14 @@ def _internal_collect_with_tag(self) -> List["Row"]:
             else None,
         )
 
+    def _execute_and_get_query_id(self) -> str:
```

Comment:
Nit: Better add a comment here it is for Table SP. It's internal but not called by other APIs so people may wonder why we need it.

Reply:
added